### PR TITLE
Update to new PF + babel config

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

**Comment by ryelo** [comment]
@karelhala any thoughts on what I should do about that test failure?

File: package.json [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
     "@babel/runtime": "^7.8.7",
-    "@data-driven-forms/pf4-component-mapper": "^2.0.8",
-    "@data-driven-forms/react-form-renderer": "^2.0.8",
-    "@patternfly/patternfly": "^2.68.3",
-    "@patternfly/react-core": "3.140.11",
-    "@patternfly/react-table": "^2.27.11",
-    "@redhat-cloud-services/frontend-components": "1.0.8",
-    "@redhat-cloud-services/frontend-components-notifications": "^1.0.1",
-    "@redhat-cloud-services/frontend-components-utilities": "1.0.0",
+    "@data-driven-forms/pf4-component-mapper": "2.5.0-beta.1",
```
Comment (Hyperkid123) [inline package.json:7]: there is 2.5 stable release

**Comment by Hyperkid123** [comment]
I will check the error locally it should not be failing

**Comment by Hyperkid123** [comment]
@ryelo @karelhala 

Right. The test is wrong. The form options object come from different context not the `Form`. You can either add it to the test or test the component inside the renderer.

Examples of both options can be found here: https://data-driven-forms.org/testing#testingcustomcomponents

Here is a diff
```diff
diff --git a/src/SmartComponents/FormComponents/DataListLayout.js b/src/SmartComponents/FormComponents/DataListLayout.js
index 756b659..b499406 100644
--- a/src/SmartComponents/FormComponents/DataListLayout.js
+++ b/src/SmartComponents/FormComponents/DataListLayout.js
@@ -52,6 +52,7 @@ DataListRow.propTypes = {
 const DataListLayout = (props) => {
     const { sections, label, name } = useFieldApi(props);
     const formOptions = useFormApi();
+    console.log({ formOptions });
     return (
         <DataList aria-label={ label || name } { ...props }>
             {sections.map(({ label, fields }, key) => {
diff --git a/src/SmartComponents/FormComponents/DataListLayout.test.js b/src/SmartComponents/FormComponents/DataListLayout.test.js
index aaeca2c..67507bc 100644
--- a/src/SmartComponents/FormComponents/DataListLayout.test.js
+++ b/src/SmartComponents/FormComponents/DataListLayout.test.js
@@ -2,21 +2,28 @@ import React from 'react';
 import { mount } from 'enzyme';
 import toJson from 'enzyme-to-json';
 import DataListLayout from './DataListLayout';
-import { Form } from '@data-driven-forms/react-form-renderer';
-describe('DataListLayout checkbox tests', () => {
+import { Form, RendererContext } from '@data-driven-forms/react-form-renderer';
 
+describe('DataListLayout checkbox tests', () => {
     it('should render correctly', () => {
         const wrapper = mount(
             <Form onSubmit={ () => undefined }>
-                {(props) => (<DataListLayout
-                    label="test label"
-                    sections={ [{ label: 'test', fields: [{ fields: []}]}] }
-                    formOptions={ {
-                        renderForm: () => 'test'
-                    } }
-                    clearedValue
-                    { ...props }
-                />)}
+                {(props) => (
+                    <RendererContext.Provider value={ {
+                        formOptions: {
+                            renderForm: () => null
+                        }} }>
+                        <DataListLayout
+                            label="test label"
+                            sections={ [{ label: 'test', fields: [{ fields: []}]}] }
+                            formOptions={ {
+                                renderForm: () => 'test'
+                            } }
+                            clearedValue
+                            { ...props }
+                        />
+                    </RendererContext.Provider>
+                )}
             </Form>
 
         );

```

**Comment by ryelo** [comment]
Updated! Thanks @Hyperkid123

**Comment by ryelo** [comment]
Actually let me update PF again here before merge

**Comment by Hyperkid123** [comment]
Still not working eh? :thinking:

**Comment by ryelo** [comment]
It should be. Travis is reporting the wrong status

**Comment by Hyperkid123** [comment]
I feel uneasy seeing the red icon :D i will close it and open

**Pull request merged**